script/hassfest/quality_scale.py

```diff
@@ -65,6 +65,1320 @@
     "unique-config-entry",
 ]
 
+INTEGRATIONS_WITHOUT_QUALITY_SCALE_FILE = [
+    "3_day_blinds",
+    "abode",
+    "acaia",
+    "accuweather",
+    "acer_projector",
+    "acmeda",
+    "acomax",
+    "actiontec",
+    "adax",
+    "adguard",
+    "ads",
+    "advantage_air",
+    "aemet",
+    "aep_ohio",
+    "aep_texas",
+    "aftership",
+    "agent_dvr",
+    "air_quality",
+    "airly",
+    "airnow",
+    "airq",
+    "airthings",
+    "airthings_ble",
+    "airtouch4",
+    "airtouch5",
+    "airvisual",
+    "airvisual_pro",
+    "airzone",
+    "airzone_cloud",
+    "aladdin_connect",
+    "alarm_control_panel",
+    "alarmdecoder",
+    "alert",
+    "alexa",
+    "alpha_vantage",
+    "amazon_polly",
+    "amberelectric",
+    "ambient_network",
+    "ambient_station",
+    "amcrest",
+    "amp_motorization",
+    "ampio",
+    "analytics",
+    "analytics_insights",
+    "android_ip_webcam",
+    "androidtv",
+    "androidtv_remote",
+    "anel_pwrctrl",
+    "anova",
+    "anthemav",
+    "anthropic",
+    "anwb_energie",
+    "aosmith",
+    "apache_kafka",
+    "apcupsd",
+    "api",
+    "appalachianpower",
+    "apple_tv",
+    "application_credentials",
```

Comment:
todo: exclude core components, do we by any chance have a good list for these in hasstfest?